views/js/e2e/interactions/order/fixtures/base64OrderInteractionTest.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
This can be deleted, because I did the test in another ticket

https://github.com/oat-sa/extension-tao-testqti/blob/feature/NEX-360_e2e-test-for-order-and-inline-interactions/views/js/e2e/runner/interactions/order/order.spec.js